src/components/app/MainContent.vue

```diff
@@ -28,27 +28,30 @@ import ConfigEditor from './ConfigEditor.vue';
 import { LogEntry } from '../shared/log/types';
 import LogsEventEmitter from './LogsEventEmitter';
 
-const colors = {
-  success: 'green',
-  failed: 'red',
-  unknown: null
+const statusToColor = {
```

Comment:
I think an enum is cleaner, instead of using the string constants every where you need to reference it. But up to you. Approved

Reply:
I removed to `enum` because you always have to write `enumName.ENUM_VALUE`.

If it wasn't TS I would never use the string constant. But they are not just a string, because if you put a wrong string it will be a compilation issue.